src/_includes/author.html

```diff
@@ -1,8 +1,8 @@
-{% assign author = site.data.people[page.author] %}
+{% set author = people[author] %}
 {% if author != null %}
 <div class="author">
     <div class="avatar">
-        <img itemprop="image" class="avatar-image avatar-image--small" src="{{author.avatar}}" alt="">
+        <img itemprop="image" class="avatar-image avatar-image--small" width="80" src="{{author.avatar}}" alt="">
```

Comment:
Added `width="80"` to the `author` image, due to GitHub ignoring `?s=80` when the user has the default avatar.